bash/01_dataloading.sh

```
@@ -23,11 +24,11 @@ wait
 rm -rf .library
 
 # Generate source_data_versions table
-psql $BUILD_ENGINE -1 -c "
+psql $BUILD_ENGINE -v ON_ERROR_STOP=1 -1 -c "
```

We should incorporate this as we do maintenance tasks. This continues to be a problem with our pipelines where we have this silent failure